README.md

```diff
@@ -1,22 +1,52 @@
 # VirtoCommerce.Cart
-VirtoCommerce.Cart module represents shopping cart management system. This module doesn't have any UI in VC Manager. 
 
-# Documentation
-User guide:
+## Overview
 
-Developer guide:
+VirtoCommerce.Cart module represents shopping cart management system. This module doesn't have any UI in VC Manager.
 
-# Installation
+Shopping cart module manages customers accumulated list of items, calculates a total for the order, including shipping and handling (i.e., postage and packing) charges and the associated taxes.
+
+![Cart Module](docs/media/screen-cart-module.png)
+
+![Cart Module Info](docs/media/screen-cart-module-info.png)
+
+## Features
```

Comment:
Please add a short description of each features, 1-2 paragraphs, not more for feature.

Reply:
OK